packages/node_modules/@webex/plugin-meetings/src/roap/index.js

```diff
@@ -242,7 +243,7 @@ export default class Roap extends StatelessWebexPlugin {
         roapMessage,
         correlationId: meeting.correlationId,
         locusSelfUrl: meeting.selfUrl,
-        mediaId: reconnect ? '' : meeting.mediaId,
+        mediaId: reconnect ? uuid.v4() : meeting.mediaId,
```

Comment:
I think this should only be in the sendRoapOffer,  When we get a roap answer its assumed there is a media ID already present !?

Reply:
I added this because Jeff S requested it:

> Instead of null media id on the rejoin, send a new media id ( that the client generates ), like you do for the initial create media.